tests/test_pipeline/test_policy/test_bert_for_pretraining_model.py

```diff
@@ -83,9 +83,11 @@ def check_bert_for_pretraining_policy():
     stage_manager = PipelineStageManager(pg_mesh, PP_DIM)
     rank = dist.get_rank()
 
-    model_policy = BertForPreTrainingPolicy(stage_manager, len(model.bert.encoder.layer))
+    model_policy = BertForPreTrainingPolicy(stage_manager)
+    model_policy.set_model(model)
+    model_policy.set_layers_per_stage()
     assert model_policy.layers_per_stage == [6, 6]
-    layers = model_policy.get_hold_layers(model)
+    layers = model_policy.get_held_layers()
     for layer in layers:
         print(layer)
```

Comment:
You just print these layers withou check. If we need to check something, please add assertation here. Otherwise, please remove this.